test/unit/language-extensions/table.spec.ts

```diff
@@ -212,18 +212,14 @@ describe("LuaTableDelete extension", () => {
             .expectToEqual({ table: { bar: 12 } });
     });
 
-    test.each([
-        'const foo = tableDelete({}, "foo");',
-        'const foo = `${tableDelete({}, "foo")}`;',
-        'declare function foo(arg: any): void; foo(tableDelete({}, "foo"));',
-        'const foo = [tableDelete({}, "foo")];',
-    ])("LuaTableDelete invalid use as expression (%p)", expression => {
+    test("LuaTableDelete use as expression", () => {
         util.testModule`
             declare const tableDelete: LuaTableDelete<{}, string>;
-            ${expression}
+            export const result = tableDelete({}, "foo")
```

Comment:
An additional test with a more complex expression would be nice (maybe just re-add the tests in the `.each block`).
Also If I understood correctly all TableExtensions can now be used as expression, so testing these as well would also be good.